app/models/miq_alert.rb

```diff
@@ -72,6 +73,14 @@ def set_responds_to_events
     self.responds_to_events = events unless events.nil?
   end
 
+  def validate_delay_next_evaluation
+    next_frequency = (options || {}).fetch_path(:notifications, :delay_next_evaluation)
+    evaluation_method = expression[:eval_method] if expression.kind_of?(Hash)
```

Comment:
I will wait on this one since this will likely take a different approach all together per [comment](https://github.com/ManageIQ/manageiq/pull/14318#issuecomment-288021533)